examples/nginx-centos7/s2i/bin/save-artifacts

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+#
+# S2I save-artifacts script for the 'nginx-centos7' image.
+# The save-artifacts script streams a tar archive to standard output.
+# The archive contains the files and folders you want to re-use in the next build.
+#
+# For more information see the documentation:
+#	https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md
+#
+tar cf - /usr/share/nginx/html/
```

Comment:
My concerns are:

1) when s2i build --incremental is specified, I believe the behaviour implemented will tar/untar all of /usr/share/nginx/html from the old container into /usr/share/nginx/html, then copy the new "source" over that.  So there's no straightforward way for a user to *remove* a file from /usr/share/nginx/html once it gets into a container, and the builds become impure - hardly exemplary behaviour.

2) partially because of 1), the example as it stands confounds more than illuminates.  If I were trying to learn how to persist C object files or a Maven cache across incremental builds, I think this example would do the opposite of helping me to understand the process.